backend/qemu.pm

```diff
@@ -953,7 +953,7 @@ sub start_qemu {
         $self->select_console({testapi_console => 'sut'});
     }
     catch {
-        if (!raw_alive) {
+        if (!$self->raw_alive) {
```

Comment:
The commit says `qemu alive status was never tested` but I don't see any new tests here

Reply:
See? This is why I had this commit as part of my signatures PR as the problem was only showing up from there. Calling `raw_alive` instead of `$self->raw_alive` means that there is an actual change of parameters and as now `$self` is required in https://github.com/os-autoinst/os-autoinst/pull/1790 as first parameter we have that covered by tests. You still have the choice to approve this PR or #1790 with the change included in the commit list

Reply:
Discussed briefly. I questioned that the code is used at all, since we don't see this fail and we have no ticket about this being broken so this might be dead code. @Martchus suggests the failure with signatures tells us that the code path works. @okurz didn't see a failure even inserting a `die` in the code path and running fullstack tests.